ci-scripts/load-test.sh

```diff
@@ -27,6 +27,11 @@ kubectl apply -f "$TEST_PIPELINE"
 
 info "Benchmark ${TEST_TOTAL} | ${TEST_CONCURRENT} | ${TEST_RUN}"
 before=$(date -Ins --utc)
+if [ -n "$WAIT_TIME" ]; then
+    info "Waiting to establish a baseline performance before creating PRs/TRs"
+    sleep $WAIT_TIME
+    info "Wait timeout completed"
+fi
```

Comment:
Yes, I've tried it but I was getting an older timestamp than to what I originally expected. Seems like we would need to capture the _started_ (the before variable in the script), prior to calling the `setup.sh` .
 
```
# Line A
 -f scenario/$TEST_SCENARIO/setup.sh ] && source scenario/$TEST_SCENARIO/setup.sh 
kubectl apply -f "$TEST_PIPELINE"

info "Benchmark ${TEST_TOTAL} | ${TEST_CONCURRENT} | ${TEST_RUN}"
before=$(date -Ins --utc)  # Line B
```

Basically move Line B to A, so we can capture "Baseline without PR" start timestamp.

But I'll need to check if this would have any impact on other scenarios.

Reply:
This `started` time is individual per scenario. E.g. for pure `bigbang` scenario, start of the test is the moment when we enable Chains. For `ongoing` scenario, start is when we start running PRs and so on. Now in your scenario, `started` should be ~20 minutes before we start running PRs :)

Reply:
So I would keep it in hands of scenario owner.

Reply:
Got it. It makes sense to keep the start time as per the scenario.